src/About/Newtab/Newtab.js

```diff
@@ -4,140 +4,129 @@
+import {Style, StyleSheet} from '../../Common/Style'
+import {cursor} from '../../Common/Cursor'
+import * as Unknown from '../../Common/Unknown'
 
-import * as Tiles from './Newtab/Tiles';
-import * as Wallpapers from './Newtab/Wallpapers';
+import * as Tiles from './Newtab/Tiles'
+import * as Wallpapers from './Newtab/Wallpapers'
 
+import type {Address, DOM} from 'reflex'
 
-import type {Address, DOM} from "reflex"
-
-export type Model =
-  { wallpapers: Wallpapers.Model
-  , tiles: Tiles.Model
-  }
-
-
-export type Action =
-  | { type: "Wallpapers", wallpapers: Wallpapers.Action }
-  | { type: "Tiles", tiles: Tiles.Action }
-
+export type Model = {
+  wallpapers: Wallpapers.Model,
+  tiles: Tiles.Model
+}
 
+export type Action = | {
+      type: "Wallpapers",
+      wallpapers: Wallpapers.Action
+    } | {
+      type: "Tiles",
+      tiles: Tiles.Action
+    }
 
 const WallpapersAction =
-  action =>
-  ( { type: 'Wallpapers'
-    , wallpapers: action
-    }
-  );
+  action => ({
+    type: 'Wallpapers',
+    wallpapers: action
+  })
 
 const TilesAction =
-  action =>
-  ( { type: 'Tiles'
-    , tiles: action
-    }
-  );
-
-export const init =
-  ():[Model, Effects<Action>] =>
-  {
-    const [tiles, tilesFx] = Tiles.init();
-    const [wallpapers, wallpaperFx] = Wallpapers.init();
-    return (
-      [
-        { wallpapers
-        , tiles
-        }
-      , Effects.batch
-        ( [ tilesFx.map(TilesAction)
-          , wallpaperFx.map(WallpapersAction)
-          ]
-        )
-      ]
-    );
-  }
+  action => ({
+    type: 'Tiles',
+    tiles: action
+  })
+
+export const init = ():[Model, Effects<Action>] => {
+  const [tiles, tilesFx] = Tiles.init()
+  const [wallpapers, wallpaperFx] = Wallpapers.init()
+  return ([
+    {
+      wallpapers,
+      tiles
+    },
+    Effects.batch([
+      tilesFx.map(TilesAction),
+      wallpaperFx.map(WallpapersAction)
+    ])
+  ])
+}
 
 const updateWallpapers = cursor({
   get: model => model.wallpapers,
   set: (model, wallpapers) => merge(model, {wallpapers}),
   update: Wallpapers.update,
   tag: WallpapersAction
-});
-
-const updateTiles = cursor
-  ( { get: model => model.tiles
-    , set: (model, tiles) => merge(model, {tiles})
-    , update: Tiles.update
-    , tag: TilesAction
-    }
-  );
-
-export const update =
-  (model:Model, action:Action):[Model, Effects<Action>] =>
-  ( action.type === 'Wallpapers'
-  ? updateWallpapers(model, action.wallpapers)
-  : action.type === 'Tiles'
-  ? updateTiles(model, action.tiles)
+})
+
+const updateTiles = cursor({
+  get: model => model.tiles,
+  set: (model, tiles) => merge(model, {tiles}),
+  update: Tiles.update,
+  tag: TilesAction
+})
+
+export const update = (model:Model, action:Action):[Model, Effects<Action>] => (
+  action.type === 'Wallpapers' ? updateWallpapers(model, action.wallpapers)
+  : action.type === 'Tiles' ? updateTiles(model, action.tiles)
   : Unknown.update(model, action)
-  );
+)
```

Comment:
Hmm. I believe according to standardjs it should be:

```js
( action.type === 'Wallpapers'
? updateWallpapers(model, action.wallpapers)
: action.type === 'Tiles'
? updateTiles(model, action.tiles)
: Unknown.update(model, action)
)
```

Actually if you don't mind doing it would be great to just rewrite our updates to `switch` statements (that we kind of started doing already here and there) as it is more readable and actually executes faster:

```js
switch (action.type) {
  case 'Wallpapers':
    return updateWallpapers(model, action.wallpapers)
  case 'Tiles':
    return updateWallpapers(model, action.wallpapers)
  default:
    return Unknown.update(model, action)
}
```